nodes/src/nodes/tool_git/git_repo.py

```
@@ -0,0 +1,952 @@
+        except pygit2.GitError as exc:
+            raise GitError(f'blame failed for {path!r}: {exc}') from exc
+
+        try:
+            entry = repo.head.peel(pygit2.Tree)[path]
+        except KeyError:
+            raise GitError(f'{path!r} not found in HEAD') from None
+        blob = repo.get(entry.id)
+        lines = (blob.data.decode('utf-8', errors='replace')).splitlines()
+
+        result = []
+        for i, hunk in enumerate(blame_obj):
```

Ruff (0.15.12)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)